README.md

```diff
@@ -795,6 +796,22 @@ iter, err := lo.Attempt(0, func(i int) error {
 // nil
 ```
 
+### Debounce
+creates a debounced instance that delays invoking func given until after wait milliseconds have elapsed.
+```go
+var counter uint64
+f := func() {
+    atomic.AddUint64(&counter, 1)
+}
+debounce := newDebounce(100 * time.Millisecond)
+// in each loop, func given is only invoked once time
+for i := 0; i < 10; i++ {
+    debounce(f, f, f, f, f)
+    time.Sleep(200 * time.Millisecond)
```

Comment:
In this example, the sleep time should be less than debounce time.

Reply:
In order to simplify the example, I recommend using the following function:

```
f := func() {
    println("Called once every 100ms!")
}
```

Counter is not essential. Let's keep samples as short as possible!

Reply:
Yeah, hh